packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -164,60 +169,89 @@ export const useArbTokenBridge = (
   const depositEth = async ({
     amount,
     l1Signer,
-    txLifecycle
+    txLifecycle,
+    destinationAddress
   }: {
     amount: BigNumber
     l1Signer: Signer
-    txLifecycle?: L1EthDepositTransactionLifecycle
+    txLifecycle?:
+      | L1EthDepositTransactionLifecycle & L1ContractCallTransactionLifecycle
```

Comment:
NIT: What's the reason for the leading `|` ? Is that a lint-correction?

Reply:
Correct